packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -158,27 +158,35 @@ const main = async () => {
 
     managedRuntimeCommand('save-credentials')
         .description(`save API credentials for Managed Runtime`)
-        .requiredOption(
-            '-u, --user <email>',
-            'the e-mail address you used to register with Managed Runtime',
-            (val) => {
-                if (!validator.isEmail(val)) {
-                    throw new program.InvalidArgumentError(`"${val}" is not a valid email`)
-                } else {
-                    return val
+        .addOption(
+            new program.Option(
+                '-u, --user <email>',
+                'the e-mail address you used to register with Managed Runtime',
+                (val) => {
+                    if (!validator.isEmail(val)) {
+                        throw new program.InvalidArgumentError(`"${val}" is not a valid email`)
+                    } else {
+                        return val
+                    }
                 }
-            }
+            )
+                .env('MRT_USER')
+                .makeOptionMandatory()
         )
-        .requiredOption(
-            '-k, --key <api-key>',
-            `find your API key at https://runtime.commercecloud.com/account/settings`,
-            (val) => {
-                if (typeof val !== 'string' || val === '') {
-                    throw new program.InvalidArgumentError(`"api-key" cannot be empty`)
-                } else {
-                    return val
+        .addOption(
+            new program.Option(
+                '-k, --key <api-key>',
+                `find your API key at https://runtime.commercecloud.com/account/settings`,
+                (val) => {
+                    if (typeof val !== 'string' || val === '') {
+                        throw new program.InvalidArgumentError(`"api-key" cannot be empty`)
+                    } else {
+                        return val
+                    }
                 }
-            }
+            )
+                .env('MRT_API_KEY')
```

Comment:
```suggestion
                .env('MRT_API_KEY')
                .conflicts('credentialsFile')
```
Prevents the two from being used together.

Reply:
In this case we don't want them to conflict as it's the save command (all three should be supported together). I'll add conflicts to the other ones